files/image_config/copp/copp_cfg.j2

```diff
@@ -91,6 +91,10 @@
 		    "trap_ids": "ip2me",
 		    "trap_group": "queue1_group1"
 	    },
+	    "eapol": {
```

Comment:
change name to eapol -> macsec, then accordingly to prince, this trap will not be installed until the feature is enabled.

Reply:
@byu343, can you do that? then we can merge.